service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/CreateGcsBucketStep.java

```diff
@@ -56,6 +56,10 @@ public StepResult doStep(FlightContext flightContext)
             .map(GcsApiConversions::toGcsApiRulesList)
             .orElse(Collections.emptyList()));
 
+    BucketInfo.IamConfiguration iamConfiguration =
+        BucketInfo.IamConfiguration.newBuilder().setIsUniformBucketLevelAccessEnabled(true).build();
+    bucketInfoBuilder.setIamConfiguration(iamConfiguration);
```

Comment:
In theory we should backfill existing controlled buckets to make this change.

Reply:
Since none of our resources are anywhere near production, we don't have to follow the theory :)

Reply:
I'm operating under the principle of "we have no workspaces/resources we care about persisting" for now. It's definitely possible for us to backfill existing buckets with some manual effort, though I don't think it's worth it.